LDAR_Sim/src/scheduling/follow_up_mobile_schedule.py

```diff
@@ -68,7 +71,7 @@ def get_plan_from_queue(self, site_id: str) -> FollowUpSurveyPlanner:
             else:
                 new_queue.put(prio, plan)
         self._survey_queue: PriorityQueueWithFIFO = new_queue
-        return prio, target_plan
+        return target_plan
```

Comment:
If Prio is unused here, can we avoid unpacking and typing it at all

Reply:
Prio is used to place back into queue.